docs/monitoring/08-04-send-notifications.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Send notifications to Slack
+type: Tutorials
+---
+This tutorial shows you how to configure Alertmanager to send notifications. Alertmanager supports several [notification receivers](https://prometheus.io/docs/alerting/configuration/), however, this tutorial focuses on sending notifications to Slack.
```

Comment:
Maybe let's link directly to https://prometheus.io/docs/alerting/configuration/#receiver ?

Reply:
Yes, good idea.